src/extensions/displayExtensionInfo.ts

```diff
@@ -18,33 +29,77 @@ const TASKS_ROLE = "cloudtasks.enqueuer";
 const TASKS_API = "cloudtasks.googleapis.com";
 
 /**
- * displayExtInfo prints the extension info displayed when running ext:install.
+ * Displays info about an extension version, whether it is uploaded to the registry or a local spec.
  *
- * @param extensionName name of the extension to display information about
- * @param spec extension spec
- * @param published whether or not the extension is a published extension
- */
-export async function displayExtInfo(
-  extensionName: string,
-  publisher: string,
+ * @param spec the extension spec
+ * @param extensionVersion the extension version
+ * */
+export async function displayExtensionVersionInfo(
   spec: ExtensionSpec,
-  published = false
+  extensionVersion?: ExtensionVersion,
+  latestApprovedVersion?: string,
```

Comment:
Nit: consider an params object here - it seems like it would be easy for callers to accidentally reverse the order on latestApprovedVersion and latestVersion